python/S3_PUBLIC_ACCESS_SETTINGS_FOR_ACCOUNT/S3_PUBLIC_ACCESS_SETTINGS_FOR_ACCOUNT.PY

```diff
@@ -0,0 +1,390 @@
+#
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+#
+# Created with the Rule Development Kit: https://github.com/awslabs/aws-config-rdk
+#
+
+'''
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  S3_PUBLIC_ACCESS_SETTINGS_FOR_ACCOUNT
+Description:
+  Checks whether AWS S3 public access account settings match the assigned parameters.
+Trigger:
+  Periodic : 1 hour
+Resource Type to report on:
+  AWS::::Account
+Rule Parameters:

```

Comment:
Let's have an opinionated default behavior. Then create the parameter to change it (as optional). It will be easier for Users to deploy.

Reply:
Changed mandatory parameters to default to true